MantisPhpClient.php

```diff
@@ -347,15 +363,16 @@ private function getProjectFromProjectTree( $p_project_id, $p_projects ) {
     /**
      * Checks if a user exists.
      *
-     * @param $p_user array that can contain id, name, or email.
+     * @param StdClass $p_user array that can contain id, name, or email.
```

Comment:
Is it a class or an array?  The type and comment conflict.  Also when referring to StdClass/array, it would be great if we specify what properties we care about in such object/array.